From c486da11feb3b9ea33732a1f0790f6ffe90a3aa0 Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Tue, 14 Feb 2012 17:14:40 +0800
Subject: ENGR00174423-1 i.mx6: cpu_op: code clean up

code clean up by removing the un-expected mfd/mfn/mfi setting

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 arch/arm/mach-mx6/cpu_op-mx6.c |   24 ------------------------
 1 file changed, 24 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu_op-mx6.c b/arch/arm/mach-mx6/cpu_op-mx6.c
index d724463..781e906 100644
--- a/arch/arm/mach-mx6/cpu_op-mx6.c
+++ b/arch/arm/mach-mx6/cpu_op-mx6.c
@@ -28,19 +28,11 @@ static struct cpu_op mx6_cpu_op_1G[] = {
 	{
 	 .pll_rate = 996000000,
 	 .cpu_rate = 996000000,
-	 .pdf = 0,
-	 .mfi = 10,
-	 .mfd = 11,
-	 .mfn = 5,
 	 .cpu_podf = 0,
 	 .cpu_voltage = 1225000,},
 	{
 	 .pll_rate = 792000000,
 	 .cpu_rate = 792000000,
-	 .pdf = 0,
-	 .mfi = 8,
-	 .mfd = 2,
-	 .mfn = 1,
 	 .cpu_podf = 0,
 	 .cpu_voltage = 1100000,},
 	{
@@ -63,10 +55,6 @@ static struct cpu_op mx6_cpu_op[] = {
 	{
 	 .pll_rate = 792000000,
 	 .cpu_rate = 792000000,
-	 .pdf = 0,
-	 .mfi = 8,
-	 .mfd = 2,
-	 .mfn = 1,
 	 .cpu_podf = 0,
 	 .cpu_voltage = 1100000,},
 	 {
@@ -86,19 +74,11 @@ static struct cpu_op mx6dl_cpu_op_1G[] = {
 	{
 	 .pll_rate = 996000000,
 	 .cpu_rate = 996000000,
-	 .pdf = 0,
-	 .mfi = 10,
-	 .mfd = 11,
-	 .mfn = 5,
 	 .cpu_podf = 0,
 	 .cpu_voltage = 1225000,},
 	{
 	 .pll_rate = 792000000,
 	 .cpu_rate = 792000000,
-	 .pdf = 0,
-	 .mfi = 8,
-	 .mfd = 2,
-	 .mfn = 1,
 	 .cpu_podf = 0,
 	 .cpu_voltage = 1100000,},
 	 {
@@ -117,10 +97,6 @@ static struct cpu_op mx6dl_cpu_op[] = {
 	{
 	 .pll_rate = 792000000,
 	 .cpu_rate = 792000000,
-	 .pdf = 0,
-	 .mfi = 8,
-	 .mfd = 2,
-	 .mfn = 1,
 	 .cpu_podf = 0,
 	 .cpu_voltage = 1100000,},
 	 {
-- 
1.7.9.5

